automatic/service-fabric-sdk/service-fabric-sdk.nuspec

```diff
@@ -5,7 +5,7 @@
     <!-- Read this before publishing packages to chocolatey.org: https://github.com/chocolatey/chocolatey/wiki/CreatePackages -->
     <id>service-fabric-sdk</id>
     <title>Service Fabric SDK</title>
-    <version>5.2.1235</version>
+    <version>5.2.1363</version>
```

Comment:
Can you confirm whether this change in SDK version to 1363 has any dependency on the related "service-fabric" package version 1363 or can remain at 1235?

```xml
<dependencies>
    <dependency id="service-fabric" version="8.2.1235" />
</dependencies>
```